schedule/yast/sle/flows/default_sle15sp4.yaml

```diff
@@ -1,49 +1,45 @@
 ---
-name: yast-mru-install-minimal-with-addons
-vars:
-  PATTERNS: base,minimal
-  YUI_REST_API: 1
-schedule:
+# Default ordered sequence of steps for sle15sp4 with maintenance updates
+bootloader:
   - installation/bootloader_start
+setup_libyui:
   - installation/setup_libyui
-  - '{{installsles}}'
+product_selection:
+  - installation/product_selection/install_SLES
+license_agreement:
   - installation/licensing/accept_license
+registration:
   - installation/registration/register_via_scc
+extension_module_selection:
   - installation/module_registration/register_extensions_and_modules
+add_on_product:
   - installation/add_on_product/add_maintenance_repos
   - installation/addon_products_sle
+suggested_partitioning:
   - installation/partitioning/accept_proposed_layout
+clock_and_timezone:
   - installation/clock_and_timezone/accept_timezone_configuration
+local_user:
   - installation/authentication/use_same_password_for_root
   - installation/authentication/default_user_simple_pwd
-  - installation/select_patterns
-  - '{{validate}}'
+software:
+  - installation/select_only_visible_patterns_from_top
+default_systemd_target: []
+booting:
   - installation/bootloader_settings/disable_boot_menu_timeout
-  - installation/installation_overview
-  - installation/disable_grub_timeout
+none_security:
```

Comment:
just `security` and for `software` I don't think we need to be so specific, in a default file we should include default patterns, so this change should be at the level of the schedule, but good for now, let's deal with that in follo-up ticket.